packages/create-project/test/cli_test.js

```diff
@@ -97,16 +97,23 @@ Object.keys(tests).forEach(projectName => {
       project: projectName,
       testRunner: tester || false
     });
+    const pkgPath = join(dir, 'package.json');
 
     t.truthy(dir);
-    assert.file(join(dir, 'package.json'));
+    assert.file(pkgPath);
     assert.file(join(dir, '.neutrinorc.js'));
     assert.file(join(dir, 'webpack.config.js'));
     assert.file(join(dir, '.eslintrc.js'));
 
     await lintable(t, dir);
     await buildable(t, dir);
 
+    const pkg = require(pkgPath); // eslint-disable-line import/no-dynamic-require
+
+    if ('start' in pkg.scripts) {
```

Comment:
As far as I can tell, the `@neutrinojs/library` preset is the only one that create-project does not configure a `start` script for. 

However the library preset does have prod-only code:
https://github.com/neutrinojs/neutrino/blob/1b4f92b2f4510aa71852638a7cca59993e7d21b1/packages/library/index.js#L101-L103

...so perhaps should still be tested in development mode?

Also, is it worth adding a comment saying something about "building in development mode to emulate start, until we can come up with something better"?

Reply:
Definitely. I'll add the comment.